homeassistant/components/fibaro/binary_sensor.py

```diff
@@ -14,7 +14,7 @@
 from .const import DOMAIN
 
 SENSOR_TYPES = {
-    "com.fibaro.floodSensor": ["Flood", "mdi:water", "flood"],
+    "com.fibaro.floodSensor": ["Flood", "mdi:water", BinarySensorDeviceClass.MOISTURE],
```

Comment:
Ideally (for a future PR) this should use/migrate to using a `SensorEntityDescription`.